src/main/java/ua/bookstore/online/exception/CustomGlobalExceptionHandler.java

```diff
@@ -42,22 +44,27 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
     }
 
     @ExceptionHandler(UniqueIsbnException.class)
-    protected ResponseEntity<ErrorResponseDto> handleMethodRepository(UniqueIsbnException ex) {
+    protected ResponseEntity<ErrorResponseDto> handleRepository(UniqueIsbnException ex) {
         return getResponseEntity(CONFLICT, ex.getMessage());
     }
 
     @ExceptionHandler(RegistrationException.class)
-    protected ResponseEntity<ErrorResponseDto> handleMethodRepository(RegistrationException ex) {
+    protected ResponseEntity<ErrorResponseDto> handleRegistration(RegistrationException ex) {
```

Comment:
```suggestion
    protected ResponseEntity<ErrorResponseDto> handleRegistrationException(RegistrationException ex) {
```
not critical, but I like it more. Up to you =)

Reply:
Discussed on QnA.